doc/release-notes.md

```diff
@@ -103,6 +103,23 @@ Low-level Changes section below.
 Low-level changes
 =================
 
+This section describes RPC changes mainly useful for testing, mostly not
```

Comment:
This isn't true for all the text in the _low-level changes_ section. I suggest you remove this sentence.

Reply:
This sentence is copied from https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.18.0.md#low-level-changes. Since I am adding this section, I need to add the description as well.

Reply:
Changed wording

Reply:
Weird. That text is clearly wrong in the v0.18 release notes.

Reply:
Text is still not correct. The changes described in the section are not for testing only. Please just remove the sentence.

Reply:
Removed for now